crates/sui-bcs-json/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+version = "0.0.0"
+name = "sui-bcs-json"
+edition = "2021"
+# version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+crate-type = ["cdylib", "staticlib", "rlib"]
+
+[workspace]
+
+[dependencies]
+sui-types = {path = "../sui-types"}
```

Comment:
We'll have to deal with these before we merge it. For now, it makes it quicker to compile & test.

Reply:
Sorry, what I meant was to import them via `workspace`.